source/_docs/scripts/conditions.markdown

```diff
@@ -436,7 +436,7 @@ Within an automation, template conditions also have access to the `trigger` vari
 ### Template condition shorthand notation
 
 The template condition has a shorthand notation that can be used to make your scripts and automations shorter. <br>
-It needs to be wrapped in quotation marks (```""```) unless using folding (```>```) where it must not use quotation marks. <br>
+It needs to be wrapped in quotation marks (```""```) unless using folding (```>```) where it must not use quotation marks. When wrapping in outside double quotation marks ensure any inner quotation marks are of the single (```'```) variant. <br>
```

Comment:
I think this whole thing needs to go, it has nothing to do with the functionality in this chapter.

Reply:
See my comment in the other PR: https://github.com/home-assistant/home-assistant.io/pull/24950. Yes, beginners mistakes but there are many beginners, no?